pkg/reading/status.go

```diff
@@ -0,0 +1,10 @@
+package reading
+
+type Status struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type Status should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)